plugins/slicer/MONAILabel/MONAILabelLib/client.py

```diff
@@ -351,10 +351,10 @@ def send_response(conn, content_type="application/json"):
                 logging.debug(f"Response FILES: {files.keys()}")
                 return response.status, form, files
             else:
-                return response.status, response.read(), None
+                return response.status, response.read(), content_disposition
 
         logging.debug("Reading status/content from simple response!")
-        return response.status, response.read(), None
+        return response.status, response.read(), content_disposition
```

Comment:
The 3rd tuple in the current returned value is files.. so you can add 4th tuple to send all the response headers (instead of just content disposition).. and fetch the content disposition from the list of response headers


```python
        if "multipart" in response_content_type:
            if response.status == 200:
                form, files = MONAILabelUtils.parse_multipart(response.fp if response.fp else response, response.msg)
                logging.debug(f"Response FORM: {form}")
                logging.debug(f"Response FILES: {files.keys()}")
                return response.status, form, files, response.getheaders()
            else:
                return response.status, response.read(), None, response.getheaders()

        logging.debug("Reading status/content from simple response!")
        return response.status, response.read(), None, response.getheaders()

```

Reply:
This is correct, thanks, the multipart function used the 3rd tuple for returning files. I've recover the 3rd and added 4th for sending response headers.

Reply:
> This is correct, thanks, the multipart function used the 3rd tuple for returning files. I've recover the 3rd and added 4th for sending response headers.  All returning variables using send_response are modified. 
Let me know if this is a decent solution.